pages/authCallbackAzure/authCallbackAzure.js

```diff
@@ -32,7 +32,7 @@ router.all('/', function(req, res, next) {
                 authn_provider_name: 'Azure',
             };
             var pl_authn = csrf.generateToken(tokenData, config.secretKey);
-            res.cookie('pl_authn', pl_authn, {maxAge: 24 * 60 * 60 * 1000});
+            res.cookie('pl_authn', pl_authn, {maxAge: config.authnCookieMaxAgeMin});
```

Comment:
The `maxAge` parameter is in milliseconds, but we are passing a value in minutes?

Reply:
Ah oops, so use something like `config.authnCookieMaxAgeMin * 60 * 1000`? Or use a millisecond var?

Reply:
I have no preference either way

Reply:
Changed to `authnCookieMaxAgeMilliseconds`